docs/core/building/index.md

```diff
@@ -0,0 +1,100 @@
+```
+
+The previous example showed how to build the `Debug` flavor, which has development time checks (asserts). To build the release (full speed) flavor, do the following:
+
+```bat 
+    .\build release skiptests
+```
+
+You can find more build options with build by using the -? or -help qualifier.   
+
+### Using Your Build
+
+The build places all of its generated files under the `bin` directory at the base of the repository.
+There is a *bin\Log* directory that contains log files generated during the build (Most useful when the build fails).
+The actual output is placed in a *bin\Product\[platform].[CPU architecture].[build type]* directory, such as *bin\Product\Windows_NT.x64.Release*.
+
+While the 'raw' output of the build is sometimes useful, normally you are only interested in the NuGet packages, which are placed in the `.nuget\pkg` subdirectory of the above output directory.
+
+There are two basic techniques for using your new runtime:
+
+ 1. **Use dotnet.exe and NuGet to compose an application**.
+    See [Using Your Build](https://github.com/dotnet/coreclr/blob/master/Documentation/workflow/UsingYourBuild.md) for instructions on creating a program that uses your new runtime by using the NuGet packages you just created and the 'dotnet' command-line interface (CLI).  This is the expected way non-runtime developers are likely to consume your new runtime.    
+
+ 2. **Use corerun.exe to run an application using unpackaged Dlls**.
+    This repository also defines a simple host called corerun.exe that does NOT take any dependency on NuGet.
+    This host has to be told where to get the necessary DLLs you actually use, and you have to manually gather them together.
+    This is the technique that all the tests in the repo use, and is useful for quick local 'edit-compile-debug' loop (for example preliminary unit testing).
+    See [Executing .NET Core Apps with CoreRun.exe](https://github.com/dotnet/coreclr/blob/master/Documentation/workflow/UsingCoreRun.md) for details on using this technique.
+
+## Build the CLI from source
+
+The source code for the .NET Core CLI can be found in [the `dotnet/cli` repository on GitHub](https://github.com/dotnet/cli/).
+
+In order to build .NET CLI, you need the following installed on you machine.
+
+* Windows & Linux:
+    - git on the PATH
+* macOS:
+    - git on the PATH
+    - Xcode
+    - OpenSSL
+
+In order to build, run `build.cmd` on Windows, or `build.sh` on Linux and macOS from the root. If you don't want to execute tests, run `build.cmd /t:Compile` or `./build.sh /t:Compile`. To build the CLI in macOS Sierra, you need to set the DOTNET_RUNTIME_ID environment variable by running `export DOTNET_RUNTIME_ID=osx.10.11-x64`.
+
+### Using your build
+
+Use the `dotnet` executable from *artifacts/{os}-{arch}/stage2* to try out the newly built CLI. You can also add *artifacts/{os}-{arch}/stage2* to the PATH if you want to use the build output when invoking `dotnet` from the current console.
```

Comment:
I think it's about moving the "if" clause before the "then" clause :) 

"You can also add *artifacts/{os}-{arch}/stage2* to the PATH if you want to use the build output when invoking `dotnet` from the current console." 

->

"If you want to use the build output when invoking `dotnet` from the current console, you can add *artifacts/{os}-{arch}/stage2* to PATH"